### PR TITLE
refactor mod

### PR DESCRIPTION
**Pull request by wsor4035**
what it says on the tin

* [x] refactor
* [x] cross test
* [x] docs?

### CODE REVIEW

**Comment by Kazooo100** [comment]
farlands reloaded materials should have bowl check if xfarming/farming redo work with it as farlands reloaded has no bowl by default

**Comment by wsor4035** [comment]
> farlands reloaded materials should have bowl check if xfarming/farming redo work with it as farlands reloaded has no bowl by default

it did not work that way before, outside of the scope of this pr

**Review by BuckarooBanzay — APPROVED** [review]
looks a lot cleaner :+1:

File: src/materials/mineclonia.lua [hunk 1]
```diff
@@ -0,0 +1,81 @@
+--note this file handles mineclonia, mineclone2, and its rename voxellibre
```
Comment (Niklp09) [inline src/materials/mineclonia.lua:1]: It's `voxelibre` afaik :-P

Comment (wsor4035) [reply src/materials/mineclonia.lua]: https://github.com/mt-mods/xcompat/pull/22/commits/4dc905ae5d5ee62fb6d974246e67376257e0130f

**Pull request merged**